pkgs/stdenv/adapters.nix

```diff
@@ -335,6 +335,25 @@ rec {
             }
       );
 
+  useWildLinker =
+    stdenv:
+    if !stdenv.targetPlatform.isLinux then
+      throw "Wild only supports Linux targets"
+    else
+      let
+        bintools = stdenv.cc.bintools.override {
+          extraBuildCommands = ''
+            wrap ld.wild ${../build-support/bintools-wrapper/ld-wrapper.sh} ${pkgs.buildPackages.wild}/bin/wild
+            wrap ${stdenv.cc.bintools.targetPrefix}ld.wild ${../build-support/bintools-wrapper/ld-wrapper.sh} ${pkgs.buildPackages.wild}/bin/wild
+            wrap ${stdenv.cc.bintools.targetPrefix}ld ${../build-support/bintools-wrapper/ld-wrapper.sh} ${pkgs.buildPackages.wild}/bin/wild
+          '';
+        };
```

Comment:
> Wild doesn't need to be wrapped

To be clear, this is the thing I would call "wrapped wild".

CC @NixOS/nixpkgs-core @infinisil this is kind of an interesting case. To do the existing style attribute names with `by-name`, we would need to do  `pkgs/by-name/wi/wild-unwrapped/package.nix` and then make this little bit `wild = ....` (or a tiny  `pkgs/by-name/wi/wild/package.nix`. But that feels like an abuse of `by-name`.

What this makes me wonder if whether we should switch from `*-unwrapped` to `*-wrapped`, and make the base package the "unopinionated" unwrapped one.

Reply:
Mold does this
https://github.com/NixOS/nixpkgs/blob/c81871f789d41b4627e77eb5c7a4bc9fa6f4e069/pkgs/top-level/all-packages.nix#L6616-L6622

Reply:
@wolfgangwalther just to be clear my point was not tiny/big package, but that it feels a bit weird for me to have `pkgs/by-name/wild-unwrapped` by the main thing, and `pkgs/by-name/wild` by the wrapper.

If `mold` has already crossed the Rubicon from `-unwrapped` to `-wrapped`, then I highly support `wild` doing that too.